src/schema/slack/functions/_scripts/generate

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
```

Comment:
I'd recommend adding some typical bash failsafes like exiting on first error (in case someone forgot to install deno before running this), exiting on undefined variables, and exiting on piped commands failing using the [`set` builtin](http://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#The-Set-Builtin). Adding a line like `set -euo pipefail` after the hash-bang should suffice, but the big one for me is exiting on first error (`set -e`).